olm/README.MD

```diff
@@ -0,0 +1,29 @@
+# Creating new OLM artifacts
+
+Script `update-nightly-olm-files.sh` is used for that
+
+Pre-requisites
+ - [yq/https://pypi.org/project/yq](https://pypi.org/project/yq) and not [http://mikefarah.github.io/yq/](http://mikefarah.github.io/yq/)
+ 
+      $ brew install python-yq (on macOS)
```

Comment:
why call out the MacOS install instructions? I mean you could also mention `pip install yq jq` too if you want to cover another 'nix method. :)

Reply:
Also just a nitpick but should the filename be README.md to be consistent w/ the one in the root of this project?

Reply:
I can remove macOS instruction if you want

BTW why jq if only yq is required ?

Reply:
fixed

Reply:
"Before using yq, you also have to install its dependency, jq. See the jq installation instructions for details and directions specific to your platform." - https://pypi.org/project/yq/